source/GooglePlayServicesNuGetReadMe.cshtml

```diff
@@ -0,0 +1,166 @@
+https://github.com/xamarin/GooglePlayServicesComponents/tree/main/samples
+
+## Key Features
+
+<!-- 
+    The key features of this package 
+-->
+
+## Main Types
+
+<!-- 
+    The main types provided in this library 
+-->
+
+## Addtional Documentation
+
+<!--
+    Provide links to more resources: List links such as detailed documentation, tutorial videos, blog 
+    posts, or any other relevant documentation to help users get the most out of your package
+-->
+### What is Google.Play.Services, Firebase and ML Kit (GPS-FB-MLKit) 
+
+
+Google Play services powers a broad set of APIs and services on Android to help you build your app, 
+enhance privacy and security, engage users, and grow your business.
+
+SDKs powered by Google Play services are backward-compatible to Android 5.0, so you can reach over 
+3 billion active Android devices worldwide.
+
+Google Play services receive automatic updates—independent of OS, OEM, or app updates—so your users 
+receive new features and bug fixes more quickly.
+
+https://developers.google.com/android
+
+
+Google Play services powers a broad set of APIs and services on Android to help you build your app, 
+enhance privacy and security, engage users, and grow your business.
+
+          
+https://developers.google.com/android/reference/packages
+
+Firebase is an app development platform that helps you build and grow apps and games users love. 
+Backed by Google and trusted by millions of businesses around the world.
+
+
+Accelerate app development with fully managed backend infrastructure
+           
+https://firebase.google.com/products-build
+
+
+Release with confidence and monitor performance and stability
+
+https://firebase.google.com/products-release
+
+
+Boost user engagement with rich analytics, A/B testing, and messaging campaigns
+           
+https://firebase.google.com/products-engage
+
+
+ML Kit brings Google’s machine learning expertise to mobile developers in a powerful and easy-to-use 
+package. Make your iOS and Android apps more engaging, personalized, and helpful with solutions that 
+are optimized to run on device.
+
+ML Kit’s processing happens on-device. This makes it fast and unlocks real-time use cases like processing 
+of camera input. It also works while offline and can be used for processing images and text that need to 
+remain on the device.
+
+https://developers.google.com/ml-kit
+
+
+## Related Packages
+
+<!-- 
+    The related packages associated with this package 
+-->
+
+https://github.com/xamarin/GooglePlayServicesComponents/blob/main/docs/artifact-list.md
+
+https://github.com/xamarin/GooglePlayServicesComponents/blob/main/docs/artifact-list-with-versions.md
+
+https://www.nuget.org/packages?q=Xamarin.GooglePlayServices
+
+https://www.nuget.org/packages?q=Xamarin.Firebase
+
+https://www.nuget.org/packages?q=Xamarin.MLKit
+
+## Feedback & Contributing
+
+<!-- 
+    How to provide feedback on this package and contribute to it 
+-->
+https://github.com/xamarin/AndroidX/issues
```

Comment:
Probably don't need a link to the AndroidX issues.

Reply:
true. Removed